tests/manage/z_cluster/nodes/test_node_replacement_reactive_aws_ipi.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import pytest
+import random
+from ocs_ci.framework.testlib import (
+    tier4, tier4b, ManageTest, aws_platform_required,
+    ipi_deployment_required, ignore_leftovers)
+from ocs_ci.ocs import machine
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from tests.helpers import wait_for_resource_state
+from tests.sanity_helpers import Sanity
+from tests.helpers import get_worker_nodes
+from ocs_ci.ocs import ocp
+
+log = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+@tier4
+@tier4b
+@aws_platform_required
+@ipi_deployment_required
+class TestNodeReplacement(ManageTest):
+    """
+    Knip-894 Node replacement - AWS-IPI-Reactive
```

Comment:
please add polarion test id

Reply:
This test script can handle 1AZ, 3AZ and failures of combination power off and network failure.
Unsure on how to add the test case ID here since in polarion we only have a single generic test case.

Reply:
This is indeed a new thing. We are not used to creating test permutations based on IPI or UPI.
@PrasadDesala please decide whether to create a polarion TC for each deployment type based on if the test steps are fundamentally different between IPI and UPI

Reply:
Created polarion test cases for each failure and volume combination .